_includes/app/txn-sample.js

```diff
@@ -84,15 +84,20 @@ function transferFunds(client, from, to, amount, next) {
     var acctBal = results.rows[0].balance;
     if (acctBal >= amount) {
       // Perform the transfer.
-      async.series([
+      async.waterfall([
```

Comment:
Switched from series to waterfall here, since with waterfall only the last async step has it's results sent to the final async callback.  Series returns the results of each function as separate callback parameters.

With series the outer function would need to have a signature along the lines of `function (err, firstUpdateResults, secondUpdateResults, selectResults) {` which I didn't think was as clear as this, where the transfer funds function returns just the updated funds.